browser_patches/firefox/juggler/TargetRegistry.js

```diff
@@ -521,7 +521,12 @@ class PageTarget {
     this._browserContext.pages.delete(this);
     this._registry._browserToTarget.delete(this._linkedBrowser);
     this._registry._browserBrowsingContextToTarget.delete(this._linkedBrowser.browsingContext);
-    helper.removeListeners(this._eventListeners);
+    try {
+      helper.removeListeners(this._eventListeners);
```

Comment:
this will fail to remove event listeners from anything after this._linkedBrowser. Does that matter?

Reply:
Not today, since linkedBrowser is the last one. Unfortunately, I don't see a better alternative. Doing a generic try/catch around each event listener in `Helper.removeListeners` could mask other problems in the future, so it seems like limiting the issues to this particular place is better. WDYT?

Reply:
Let's dump the error and not worry about it.